frontend/src/metabase/dashboard/components/DashboardGrid.jsx

```diff
@@ -366,8 +377,12 @@ class DashboardGrid extends Component {
         layouts={layouts}
         breakpoints={GRID_BREAKPOINTS}
         cols={GRID_COLUMNS}
-        width={width}
-        margin={{ desktop: [6, 6], mobile: [6, 10] }}
+        width={dashboard.is_app_page ? DATA_APPS_MAX_WIDTH : width}
```

Comment:
At some point I feel like we're going to want to just fork the `DashboardGrid` component. I think that'd make it easier to reason about.